crates/fuel-streams-core/src/utxos/mod.rs

```diff
@@ -0,0 +1,16 @@
+pub mod subjects;
+pub mod types;
+
+use serde::{Deserialize, Serialize};
+pub use subjects::*;
+
+use crate::prelude::*;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Utxo(pub Option<Vec<u8>>);
```

Comment:
Nit: Having a small doc for the inner type would also be nice.

Reply:
yep, will happily add it once we decide on the payload